soak_tests/README.md

```diff
@@ -0,0 +1,51 @@
+# Soak Tests
+These tests are designed to simulate typical production load, over a continuous availability period, to validate system behavior under production use.
```

Comment:
I think it's worth mentioning here or in the `Running` section that these run against `${cloudant_username}.cloudant.com`